src/backend/joanie/core/models/accounts.py

```diff
@@ -59,3 +60,44 @@ def save(self, *args, **kwargs):
     def get_full_address(self):
         """Full address to display"""
         return f"{self.address}\n{self.postcode} {self.city}\n{self.country.name}"
+
+
+class CreditCard(models.Model):
+    """Credit card model to allow to save user credit cards"""
+
+    uid = models.UUIDField(
+        default=uuid.uuid4, unique=True, editable=False, db_index=True
+    )
+    name = models.CharField(_("name"), max_length=50)
+    # token returned by service payment backend
+    token = models.CharField(_("token"), max_length=128)
+    expiration_date = models.DateField(_("expiration date"), db_index=True)
```

Comment:
may not be the best format to save this info as we don't have a day but just a month and year.
Maybe a CharField with a regex of the type "[0-9]{2}/[0-9]{2}" is enough?

Reply:
I save the last day of month to easily manage cards expiration
https://github.com/openfun/joanie/pull/38/files#diff-2fec5570001681150ded72601ca01ace7663cc59e096e97fa97ce8ef63ebfa7eR151